src/Bicep.Cli/Commands/FormatCommand.cs

```diff
@@ -1,49 +1,22 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using System.IO.Abstractions;
 using Bicep.Cli.Arguments;
 using Bicep.Cli.Helpers;
 using Bicep.Core.Configuration;
 using Bicep.Core.Diagnostics;
-using Bicep.Core.Features;
 using Bicep.Core.FileSystem;
 using Bicep.Core.Parsing;
-using Bicep.Core.PrettyPrint;
-using Bicep.Core.PrettyPrint.Options;
 using Bicep.Core.PrettyPrintV2;
-using Microsoft.Extensions.Logging;
 
 namespace Bicep.Cli.Commands;
 
-public class FormatCommand : ICommand
+public class FormatCommand(IOContext io, IFileResolver fileResolver, IFileSystem fileSystem, IConfigurationManager configurationManager) : ICommand
```

Comment:
Worth breaking args onto separate lines?

Reply:
Yeah makes sense. I wish the C# formatter can implement the same algorithm :p

Btw, this is a bad example of primary constructor conversion - I mistakenly used the wrong code fix option which removes the read-only fields without noticing it 🤦‍♂️. Will restore them in a new commit.

Reply:
Ended up reverting it to the traditional constructor syntax, as my preference for the primary constructor has diminished...A limitation I just realized is the potential for confusion between class fields and primary constructor parameters - omitting `this.` can mistakenly reference a constructor argument rather than the intended read-only class field. Personally, I'll pause the use of primary constructor until the C# team implements [the readonly parameters proposal](https://github.com/dotnet/csharplang/blob/ee2c62fbacf3e84457198624c71abdca326c2cd8/proposals/readonly-parameters.md).